domains/kitten.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "a cool website",
+  "repo": "https://github.com/KittensAreDaBest/is-a-dev",
```

Comment:
This should be the repo for the website.

Reply:
> This should be the repo for the website.

wym by that

Reply:
ah i see now. should be fixed